components/social/org.wso2.carbon.social.sql/src/main/java/org/wso2/carbon/social/sql/SQLActivityBrowser.java

```diff
@@ -398,6 +411,39 @@ public JsonObject getTopComments(String targetId, int likes)
 		}
 	}
 
+	public JsonObject getUserComment(String userId, String targetId) throws SocialActivityException {
+		JsonObject userComment = new JsonObject();
+		Connection connection = null;
+		PreparedStatement statement;
+		ResultSet resultSet;
+		String errorMsg = "Unable to retrieve comment for the user : " + userId;
+
+		try {
+			if (log.isDebugEnabled()) {
```

Comment:
Thank you, I have fixed this issue in following [commit](https://github.com/wso2/carbon-store/pull/643/commits/358a87f6fa0ffb067480daf1fe3e463bed87080e)